pages/register/consumer.vue

```diff
@@ -512,20 +511,12 @@ export default class ConsumerRegistrationPage extends mixins(BaseMixin, Validati
         if (!this.registrationRequest.rootUser) {
             return
         }
-
-        const loginRequest: LoginWithPasswordRequest = {
-            email: this.registrationRequest.rootUser.email as string,
-            password: {
-                value: this.registrationRequest.password as string,
+        return this.$router.push({
+            path: '/login/verify',
+            query: {
+                send: 'true',
+                cons: this.consumer?.id.id,
```

Comment:
will this.consumer?.id.id always be available? even after a reload?
it could cause 'cons=undefined' in url.

Reply:
Yeah, if it's not available it means the API call to createConsumer failed. And if that's failed it's caught and this function won't be called.